pyproject.toml

```diff
@@ -0,0 +1,39 @@
+[tool.poetry]
+name = "pipreqs"
+version = "0.4.13"
+description = "Pip requirements.txt generator based on imports in project"
+authors = ["Vadim Kravcenko <vadim.kravcenko@gmail.com>"]
+license = "Apache-2.0"
+readme = ["README.rst", "HISTORY.rst"]
+packages = [{include = "pipreqs"}]
+repository = "https://github.com/bndr/pipreqs"
+keywords = ["pip", "requirements", "imports"]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: Apache Software License",
+    "Natural Language :: English",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11"
+]
+
+[tool.poetry.scripts]
+pipreqs = "pipreqs.pipreqs:main"
```

Comment:
what does this do exactly? is it doing something like setting pipreqs as a cli callable script?

Reply:
Sort of. This was inherited from 'setup.py' file (see https://github.com/bndr/pipreqs/pull/409/files?short_path=60f61ab#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7L52-L56)